### ISSUE TITLE
IOS App does not work

### ISSUE THREAD

**Issue by easez88** [BeryJu/hass-auth-header#237]
I can't seem to get this to work with the IOS app. I've installed it and added the necessary items to my config. I can use authentik to authenticate in the browser with no issues. However, The iOS app is not able to sign in. The authentik prompt appears and allows me to sign in successfully, but right when it would redirect me back to the app, I get an error. This causes the first issue, unable to log in with mobile app. 

To work around this, I sign in locally first with hass ip address then I try to use use my domain name. When i do this the app appears to work as intended, but the device tracker never updates. Location updates don't trigger, manual refreshes, don't update hass. 

If I disable this plugin, everything goes back to normal. Does anyone know how to correct this issue?

TLDR:
Plugin works through browser, but cannot use with iOS mobile app.

**Comment by olekenneth** [comment]
Same for me.

**Comment by Swiftnesses** [comment]
I have the same issue (or very similar). I've thought it was working, but the app is unable to get entities or update locations when away from home - the debugging in the Home Assistant app shows "too many redirects".

SSO login works fine (even remote from inside the HA app), but the app doesn't work correctly.

Did anyone solve this?
